airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1816,11 +1816,11 @@
 - name: WooCommerce
   sourceDefinitionId: 2a2552ca-9f78-4c1c-9eb7-4d0dc66d72df
   dockerRepository: airbyte/source-woocommerce
-  dockerImageTag: 0.1.2
+  dockerImageTag: 0.2.0
```

Comment:
I'm seeing a part of the CI build phase failing with the following error:
```
LocalDefinitionsProviderTest > initializationError FAILED
    java.lang.UnsupportedOperationException: There is no seed spec for docker image airbyte/source-woocommerce:0.2.0
```
Since we haven't published the image yet and it can't be found, is this an expected failure? I see on some PRs you've bump it, but other not.

Reply:
not sure about this exact test, but I guess it is ok for other unittests to fail, as this PR is still blocked by https://github.com/airbytehq/airbyte/issues/19753